### PR TITLE
PHPC-321: Rename UTCDatetime class to UTCDateTime

### PR DESCRIPTION
**Pull request by derickr**
Depends on the PHPC-309 pull request: https://github.com/10gen-labs/mongo-php-driver-prototype/pull/48

### CODE REVIEW

File: RELEASE-0.4.0 [hunk 1]
```diff
@@ -167,7 +167,7 @@
 	* PHPC-222: WriteConcernError can cause segfaults on Ubuntu Precise 32bit
 
 2015-04-01  Hannes Magnusson  <bjori@php.net>
-	* PHPC-220: BSON\UTCDatetime broken on 32bit
+	* PHPC-220: BSON\UTCDateTime broken on 32bit
```
Comment (bjori) [inline RELEASE-0.4.0]: ?

**Comment by bjori** [comment]
Previous release notes shouldn't be modified -- at the time the classname really was UTCDatetime.
Also, not sure what those test changes were about?
Can you file a ticket for those changes so we can look into that separately, it isn't related to the classname renaming. Seems like you were debugging something unrelated?

Also, it helps a lot if you only include the specific commit for the ticket, not all your previous Pull Requests.

Anyway, its fixed now: https://github.com/10gen-labs/mongo-php-driver-prototype/commit/803f4317f1045be178eb34e178af510a25d3f654

Thanks for the ticket!

File: RELEASE-0.4.0 (hunk 1 above)
Comment (jmikola) [inline RELEASE-0.4.0]: I assume it was search/replace or retcon-ing old release logs.

**Comment by jmikola** [comment]
> Also, it helps a lot if you only include the specific commit for the ticket, not all your previous Pull Requests.

This was based on his other PR and he mentioned the dependency in the original post. It's nothing we haven't already done with 1.x development when we had inter-related PRs with a dependent merge order. Were #48 to be merged, this would get rebased and factor down to the single commit.

**Pull request closed without merging**